src/h3/_cy/h3lib.pxd

```diff
@@ -2,9 +2,27 @@ from cpython cimport bool
 from libc.stdint cimport uint64_t, int64_t, uint32_t
 
 ctypedef uint64_t H3int
-ctypedef uint32_t H3Error
 ctypedef basestring H3str
 
+# todo: extern version of this?
+cdef enum H3Error:
```

Comment:
is there a way to inform the Cython compiler of the bit width of this enum?

Reply:
Not sure yet. The confusion between the error type and the error codes was what led me to https://github.com/uber/h3/pull/613

Not sure what the most elegant solution is here, but definitely open to ideas!